### ISSUE TITLE
Font Fidelity on Mac

### ISSUE THREAD

**Issue by theruffus** [cgiesche/streamdeck-homeassistant#281]
TLDR: Just bought Elgato Stream Deck Neo, fonts look weird on a Mac, but normal on PC.

I use a Mac Mini M1 for work and a PC for gaming, after buying the Elgato Stream Deck Neo (the new white one with 8 buttons)
I connected it via USB Switcher to be able to use it on both machines.

When configuring some home automation buttons I noticed fonts in home assistants on a Mac look like "the printer running out of ink". Example below, both buttons configured the same way in the same spot, first one on a Mac, second on a PC:

Mac:
![mac_neo](https://github.com/cgiesche/streamdeck-homeassistant/assets/30386731/4db4c6d6-b4ba-4156-b9a9-7fbf6b4ec044)
PC:
![pc_neo](https://github.com/cgiesche/streamdeck-homeassistant/assets/30386731/5947bcb6-0f4d-4bbd-8236-73a535458818)

Perhaps expectedly they look in the same "printer out of ink" way in the preview in the Stream Deck app.

![Zrzut ekranu 2024-06-23 o 22 36 38](https://github.com/cgiesche/streamdeck-homeassistant/assets/30386731/7c0af731-f7d2-4157-9343-ad534fa3108d)

Maybe offering a font choice or setting some smoothing settings would fix this?

**Comment by theruffus** [comment]
I did some searching, the culprit seems to be the black stroke (strokeWidth:4) in the **plugin-780664f7.js file.**
One changed strokeWidth to 0 and the stroke to #FFF the text became crystal clear on the Mac

`class J3{constructor(V={width:144,height:144}){this.buttonRes=V,this.halfRes={width:this.buttonRes.width/2,height:this.buttonRes.height/2},this.fontSize=24,this.lineAttr={fill:"#FFF","font-family":"sans-serif","font-weight":"bold","font-size":`${this.fontSize}px`,"text-anchor":"middle",stroke:"#FFF",strokeWidth:0}`

Result:
![IMG_1855](https://github.com/cgiesche/streamdeck-homeassistant/assets/30386731/4a9b8305-132f-4e12-8e12-1b83c72044eb)

Cheers

**Comment by cgiesche** [comment]
Hi and thank you for this finding. I am currently (slowly but surely) working on a refresh of the rendering logic. I will add your changes to make the font more readable :)

**Comment by Notrial** [comment]
Not sure if this was asked before but it would be awesome if we could have text formatting options for custom labels as well, similar to the title options on screenshot below:

<img width="358" alt="image" src="https://github.com/user-attachments/assets/dcde85ec-a133-4a93-8db6-7a190ac00986">
